curator-test-zk34/pom.xml

```diff
@@ -160,7 +160,7 @@
                         <dependency>org.apache.curator:curator-framework</dependency>
```

Comment:
Didn't happen in this PR, but this file has 2 curator-recipes dependencies

Reply:
Is there a reason why we don't have curator-x-async dependency here?

Reply:
It's not tested for zk34 compatibility. We don't make any guarantees that that module works with zk 3.4